cfgov/ask_cfpb/models/django.py

```diff
@@ -547,6 +546,10 @@ def has_live_page(self):
                 return True
         return False
 
+    # def sync_related_questions(self, page):
```

Comment:
Can this commented out code be deleted for now?

Reply:
Gone!